### ISSUE TITLE
NER Pipeline returns null

### ISSUE THREAD

**Issue by Realvincentyuan** [huggingface/transformers#3292]
# 🐛 Bug

## Information

Model I am using (NER Pipeline):

Language I am using the model on (English):

The problem arises when using:
* [x] the official example scripts: (give details below)
* [ ] my own modified scripts: (give details below)

The tasks I am working on is:
* [ ] an official GLUE/SQUaD task: (give the name)
* [x] my own task or dataset: (give details below)

## To reproduce

```
from transformers import pipeline
# Allocate a pipeline for named entity recognition
nlp = pipeline('ner')
nlp(['We are very happy to include pipeline into the transformers repository.'])

```
**returns null.**

## Expected behavior

Expect to have the named entity label for each token. But it returns null.

## Environment info

     
- `transformers` version: 2.5.1
- Platform: Jupyter lab
- Python version: 3.6.1
- PyTorch version (GPU?): CPU-1.4.0
- Tensorflow version (GPU?): CPU-1.15.0
- Using GPU in script?: No
- Using distributed or parallel set-up in script?: No

**Comment by LysandreJik** [comment]
By "it returns null" you mean it returns an empty array? That's because it didn't identify any named entity in your sequence.
